src/mame/video/k051960.cpp

```diff
@@ -428,7 +434,12 @@ void k051960_device::k051960_sprites_draw( bitmap_ind16 &bitmap, const rectangle
 		code = m_ram[offs + 2] + ((m_ram[offs + 1] & 0x1f) << 8);
 		color = m_ram[offs + 3] & 0xff;
 		pri = 0;
-		shadow = color & 0x80;
+		if (m_shadow_config & 0x04)
+			shadow = 0;
+		else if (m_shadow_config & 0x02)
+			shadow = 1;
+		else
+			shadow = color & 0x80;
```

Comment:
It would be clearer to express this as either:
```
    shadow = !(m_shadow_config & 0x04) && ((m_shadow_config & 0x02) || (color & 0x80));
````
or
```
    shadow = !BIT(m_shadow_config, 2) && (BIT(m_shadow_config, 1) || BIT(color, 7));
````

It should really be a `bool` – it’s an `int` inherited from pre-C++ MAME.  (The other things declared in the same line could also do with tightening up scope.  It’s all leftover from the pre-C99 days when locals could only be declared at the start of a compound statement.)